lib/tasks/default_admin_set.rake

```diff
@@ -3,24 +3,25 @@ namespace :hyrax do
   namespace :default_admin_set do
     desc "Create the Default Admin Set"
     task create: :environment do
-      id = AdminSet.find_or_create_default_admin_set_id
-      # I have found that when I come back to a development
-      # environment, that I may have an AdminSet in Fedora, but it is
-      # not indexed in Solr.  This remediates that situation by
-      # ensuring we have an indexed AdminSet
-      AdminSet.find(id).update_index
-      if Hyrax::PermissionTemplate.find_by(source_id: id)
+      id = Hyrax::AdminSetCreateService.find_or_create_default_admin_set.id
+      if Hyrax::PermissionTemplate.find_by(source_id: id.to_s)
         puts "Successfully created default admin set"
       else
         warn "ERROR: Default admin set exists but it does not have an " \
           "associated permission template.\n\nThis may happen if you cleared your " \
-          "database but you did not clear out Fedora and Solr.\n\n" \
+          "database but you did not clear out metadata datasource (e.g. Fedora, Postgres) " \
+          "and Solr.\n\n" \
           "You could manually create the permission template in the rails console" \
           " (non-destructive):\n\n" \
-          "    Hyrax::PermissionTemplate.create!(source_id: AdminSet::DEFAULT_ID)\n\n" \
-          "OR you could start fresh by clearing Fedora and Solr (destructive):\n\n" \
+          "    Hyrax::PermissionTemplate.create!(source_id: Hyrax::AdminSetCreateService::DEFAULT_ID)\n\n" \
+          "OR you could start fresh by clearing the metadata datasource and Solr (destructive):\n\n" \
+          "  For ActiveFedora or Wings Valkryie adapter (default), use...\n" \
```

Comment:
do the valkyrie instructions not work (or not work efficiently?) for Wings?

it's not very important to this PR, but i was wondering.

Reply:
In `bundle exec rails console`...
```
>> conn = Hyrax.index_adapter.connection
Traceback (most recent call last):
        1: from (irb):1
NoMethodError (undefined method `connection' for #<Valkyrie::Indexing::NullIndexingAdapter:0x00007fb386b2b1c8>)
>> Hyrax.persister.wipe!
Traceback (most recent call last):
        2: from (irb):1
        1: from (irb):2:in `rescue in irb_binding'
NameError (uninitialized constant ActiveFedora::Cleaner)
>> require 'active_fedora/cleaner'
=> true
>> Hyrax.persister.wipe!
Attempted to init base path `dev`, but it already exists
=> {"responseHeader"=>{"status"=>0, "QTime"=>1}}
>> 
```

Should wings persister include `require 'active_fedora/cleaner` such that `Hyrax.persister.wipe!` can work without doing the require.  This will simplify the instructions.  For wings, it will wipe the solr that AF uses.  But it doesn't wipe the solr for `indexing_adapter` when it is writing to the valkyrie solr.

I'd like to explore this outside of this PR.  The PR is already has a lot of changes in it.